backend/src/model/challenge.rs

```diff
@@ -16,10 +18,13 @@ struct NewGameChallenge {
     ranked: bool,
     public: bool,
     tournament_queen_rule: bool,
+    color_choice: String,
     created_at: DateTime<Utc>,
 }
 
-#[derive(Identifiable, Queryable, Debug)]
+#[derive(Associations, Identifiable, Queryable, Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+#[diesel(belongs_to(User, foreign_key = challenger_uid))]
 #[diesel(table_name = game_challenges)]
 pub struct GameChallenge {
     pub id: Uuid,
```

Comment:
I advocate for `id` as the first field and then alphabetically sorted because then we are always sure the fields in `NewX` and `X`and I would like to do the same for the migrations.

Reply:
i was originally doing something similar, but ultimately i don't think it makes sense -- if you add columns to a table later down the road, diesel just appends it to the end of the schema, so we'll end up w/ non-alphabetical fields anyway